### PR TITLE
 links in html

### PR DESCRIPTION
**Pull request by irasar**
Links put in index.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,11 +1,35 @@
...
+
+    <!-- font awesome  -->
+    <script src="https://kit.fontawesome.com/f4677e3cb7.js" crossorigin="anonymous"></script>
+
+    <!-- bulma css framework -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.0/css/bulma.min.css">
+
+
+    <!-- our style sheet  -->
+    <link rel="stylesheet" href="assets/style.css">
```
Comment (Crowe828) [inline index.html]: Looks Great

File: index.html [hunk 2]
```diff
@@ -1,11 +1,35 @@
...
 </head>
+
 <body>
-    
+
+    <input class="input is-rounded" type="text" placeholder="Rounded input">
+    <a class="button is-info">
+        Search
+      </a>
```
Comment (Crowe828) [inline index.html:25]: Excellent

**Review by Crowe828 — COMMENTED** [review]
Great README!

**Review by Crowe828 — COMMENTED** [review]
Looks good

**Comment by brigil** [comment]
looking good

**Review by Crowe828 — APPROVED** [review]

**Pull request merged**